test/unit_tests/aievec_tests/i16xi16_static_sized_memref_aie-ml/gen_aie-ml.cc

```diff
@@ -0,0 +1,38 @@
+void conv2d(int16_t * restrict v1, int16_t * restrict v2, int16_t * restrict v3) {
```

Comment:
I may be misunderstanding this, but should this file be generated by the test itself? Where it says: `aie-translate -aieml=true --aievec-to-cpp -o gen_aie-ml.cc`

Reply:
Yes, it is generated by aie-translate. I put it here just for the reference.